File: DS-Algo-with-Java/Lecture-10/SmallestNumber.java
import java.util.Scanner;

public class SmallestNumber {
    public static void main(String[] args){
        try (Scanner sc = new Scanner(System.in)) {
            System.out.print("Enter the first number: ");
            int a = sc.nextInt();
            System.out.print("Enter the second number: ");
            int b = sc.nextInt();
            System.out.print("Enter the third number: ");
            int c = sc.nextInt();
            System.out.print(smallest(a, b, c));
        }
    }

    public static int smallest(int a, int b, int c){
        return Math.min(a, Math.min(b,c));
    }
}

File: DS-Algo-with-Python/Lecture-10/SmallestNumber.py
def Smallest(a, b, c):
    return min(a, min(b, c))

a = int(input("Enter 1st number: "))
b = int(input("Enter 2nd number: "))
c = int(input("Enter 3rd number: "))

print(f"The smallest number is: {Smallest(a, b, c)}")
